README.md

```diff
@@ -30,6 +30,10 @@ There are many areas we can collaborate on. In some areas work is already being
 
 - **[Conferences and Meetups](https://github.com/w3c/webcomponents-cg/labels/conferences-and-meetups)** - Organize conferences and meetups that include topics and participants from across the web components ecosystem.
 
+## Meetings
+
+The Web Components Community Group holds [a shared calendar](https://calendar.google.com/calendar/embed?src=o25bim5rvcu42mfnqilirpmp44%40group.calendar.google.com&ctz=America%2FLos_Angeles) of its meetings and events. A ics file if [available here](https://calendar.google.com/calendar/ical/o25bim5rvcu42mfnqilirpmp44%40group.calendar.google.com/public/basic.ics).
```

Comment:
```suggestion
The Web Components Community Group holds [a shared calendar](https://calendar.google.com/calendar/embed?src=o25bim5rvcu42mfnqilirpmp44%40group.calendar.google.com) of its meetings and events. A ics file if [available here](https://calendar.google.com/calendar/ical/o25bim5rvcu42mfnqilirpmp44%40group.calendar.google.com/public/basic.ics).
```

It looks like this opens the calendar to use local time, does that hold true for others? Non-Pacific Time members might like that...

Reply:
That's a bit tricky feature from Google Calendar.

I did set the default timezone of this calendar to Eastern Time, due to the perceived usage in the notes doc. The link I shared has a url parameter to provide it in my local timezone, but the calendar importing the entries will match to the users local timezone when they eventually import it.

We could expand the link to provide a list of the same with different url params representing different timezones.

Reply:
For better clarification, the current link, after the modification from the commit suggestion, will open the calendar in its default timezone, Eastern Time. I'm currently on Pacific Time, and it's not an issue for me but I'm used to it (bias).